### PR TITLE
[MINOR][EDA-1685] - Add Circle Ci in Wumpus

### PR DESCRIPTION
**Pull request by npaura-eb**
In order to upload the wumpus docker image to the ECR in AWS Sandbox, it is necessary to configure circle ci to build and upload the image and trigger its update when a change is pushed to the main branch of wumpus.   

So for that, the `build-and-upload-image` and `trigger-main-repo-update` jobs were added in the `config.yml` file inside the `.circleci` directory.

### CODE REVIEW

**Review by mbertani-eb — APPROVED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -30,9 +30,86 @@ jobs:
           name: Update CodeClimate coverage report
           command: |
               ./cc-test-reporter after-build --exit-code $?
+  build-and-upload-image:
+    docker:
+      - image: docker:20.10.14-git
+    parameters:
+      app_name:
+        type: string
+        description: Name of the application to build
```
Comment (agutierrez-eb) [inline .circleci/config.yml:39]: Is this the right description?

File: .circleci/config.yml [hunk 2]
```diff
@@ -30,9 +30,86 @@ jobs:
...
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run:
+          name: Install dependencies
+          command: |
+            apk add --no-cache py3-pip
```
Comment (agutierrez-eb) [inline .circleci/config.yml:46]: Just out of curiosity. Are we using alpine here? As far as i know, apk is the package installer of alpine.

File: .circleci/config.yml (hunk 1 above)
Comment (npaura-eb) [reply .circleci/config.yml:39]: In that step, the app_name parameter is declared without a value. Below, in the workflows jobs, the value to app_name  is assigned to edagames-wumpus.

In our case, the application to build is wumpus. Do you suggest changing the description to "wumpus app" or something like that?

```
workflows:
  main:
    jobs:
      - build-and-test
      - build-and-upload-image:
          app_name: edagames-wumpus
```

Comment (agutierrez-eb) [reply .circleci/config.yml:39]: Nice! I see. You have my 🟢

**Review by agutierrez-eb — APPROVED** [review]

File: .circleci/config.yml (hunk 2 above)
Comment (npaura-eb) [reply .circleci/config.yml:46]: Yes, the docker:20.10.14-git image that we are using has alpine as the operating system

**Pull request merged**